automations/remediate_messages_via_herd_immunity.yml

```diff
@@ -0,0 +1,8 @@
+name: "Remediate messages via herd immunity"
```

Comment:
per discussion [here](https://sublimesecurity.slack.com/archives/C07D63BFSEL/p1731943731270829?thread_ts=1731528259.775899&cid=C07D63BFSEL), this should be:

name: "Remediate user reports via herd immunity"

Reply:
thanks I'll update the Notion task too.